ecs-cli/modules/cli/local/network/setup.go

```diff
@@ -77,7 +77,7 @@ const (
 )
 
 // Setup creates a user-defined bridge network with a running Local Container Endpoints container. If the network
-// already exists or the container is already running then the operation is a no-op.
+// already exists or the container is already running then this function does nothing.
```

Comment:
Very minor nit: this change and the other one in teardown.go probably shouldn't be in the "generate mocks" commit.

If you plan to squash all changes to one commit prior to merge, then don't worry about it